src/gobexport/test.py

```diff
@@ -76,15 +82,113 @@
 _ABSOLUTE_VALUES = ["empty_lines", "first_bytes", "first_lines"] + [f"{nth}_line" for nth in _NTH.values()]
 
 # chunksize for downloading from objectstore, must be < 2gb
-_CHUNKSIZE = 500_000_000
+_CHUNKSIZE = 100_000_000
+# download files bigger then this threshold to disk
+_OFFLOAD_THRESHOLD = 500_000_000
+
+ENCODING = 'utf-8'
+TYPE_FLAT_FILE = ("plain/text", "text/csv", "application/x-ndjson")
+TYPE_CSV = ("text/csv", )
+
+# collection of unicode numbers for str characters
+WHITESPACE = {ord(c) for c in ' \t\n\r\v\f'}
+ASCII_LOWER = {ord(c) for c in 'abcdefghijklmnopqrstuvwxyz'}
+ASCII_UPPER = {ord(c) for c in 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'}
+DIGITS = {ord(c) for c in '0123456789'}
+
+
+def _safe_divide(val1: int, val2: int) -> Union[int, float]:
+    try:
+        result = val1 / val2
+        if result.is_integer():
```

Comment:
keep try clauses as small as possible. is_integer is not going to raise a zerodivision error.